README.md

```diff
@@ -1,24 +1,126 @@
-# activair
-
-## Project setup
-```
-npm install
-```
-
-### Compiles and hot-reloads for development
-```
-npm run serve
-```
-
-### Compiles and minifies for production
-```
-npm run build
-```
-
-### Lints and fixes files
-```
-npm run lint
-```
-
-### Customize configuration
-See [Configuration Reference](https://cli.vuejs.org/config/).
+# ActivAir
+###### Find fresh air for your outdoor activities.
+---
+## Table of Contents
+* [Introduction](#Introduction)
+* [Technologies](#Technologies)
+* [Deployment](#Deployment)
+* [Features](#Features)
+* [Learning Process](#Learning-Process)
+* [Authors](#Authors)
+
+## Introduction
+Last summer, as wildfires were raging across the western United States, residents who live and recreate nearby were often faced with the decision of whether to be active outdoors in the first place, and if so, where to go. Hikers, runners, cyclists, dog walkers, and more could have used an app that brings in up-to-date weather and air quality information by location to help make that decision.
+
+[ActivAir](https://github.com/alia-peterson/ActivAir) solves this problem by tapping into data from the [IQAir AirVisual API](https://www.iqair.com/us/). Users can view data for their current location, add and remove other locations, get advice on whether it's safe to recreate outdoors given the AQI for that area, and click to view trail recommendations for each location on [AllTrails](https://www.alltrails.com/). The project specifications can be found [here](https://frontend.turing.io/projects/module-3/stretch.html).
+
+### Motivation & Goals
+* Learn a new "stretch" technology independently. After learning React last month, we chose to dive into Vue.js to deepen our understanding of frameworks. We had 9 days to self-teach Vue.js and create our app.
+* Create an MVP that solves a problem for users
+* Gain more experience using external APIs
+
+---
+
+## Technologies
+JavaScript, Vue, RESTful APIs, Cypress, Local Storage, HTML, CSS
+
+---
+
+## Deployment
+TO DO [Deployment Link]()
+
+**To run locally:**
+1. Clone down this repo
+2. `npm install`
+3. `npm run start`
+
```

Comment:
not currently applicable - this is where I expect we'd want to add instructions if users needed to retrieve an API and set up that file for themselves.

Reply:
agreed!